dnf/cli/option_parser.py

```diff
@@ -280,8 +280,8 @@ def _add_general_options(self):
         general_grp.add_argument("--repofrompath", default={},
                                  action=self._SplitExtendDictCallback,
                                  metavar='[repo,path]',
-                                 help=_("label and path to additional repository,"
-                                        " can be specified multiple times."))
+                                 help=_("label and path to additional repository (same path as in "
```

Comment:
While at it, I'd fix it:
> label and path to **an** additional repository (same path as in ...

And I think it's a bit open-ended, I would also add:
> label and path to **an** additional repository **to use** (same path as in ...

Reply:
Ok, edited. Thanks.